core/src/main/scala/kafka/admin/ZkSecurityMigrator.scala

```diff
@@ -120,17 +123,18 @@ object ZkSecurityMigrator extends Logging {
   }
 }
 
-class ZkSecurityMigrator(zkUtils: ZkUtils) extends Logging {
+class ZkSecurityMigrator(zkClient: KafkaZkClient) extends Logging {
+  private val zkSecurityMigratorUtils = new ZkSecurityMigratorUtils(zkClient)
   private val futures = new Queue[Future[String]]
 
   private def setAcl(path: String, setPromise: Promise[String]) = {
```

Comment:
```suggestion
  private def setAcl(path: String, setPromise: Promise[String]): Unit = {
```

Maybe fix these void methods while you're in here?

Reply:
Updated the void methods, though we are not strictly following this style